src/actions/ticket-actions.js

```diff
@@ -482,8 +482,10 @@ export const getTickets =
       page,
       per_page: perPage,
       access_token: accessToken,
-      expand:
-        "owner,order,ticket_type,badge,badge.type,promo_code,promo_code.tags,refund_requests,refund_requests.refunded_taxes"
+      expand: "owner,badge,promo_code,promo_code.tags,refund_requests",
+      fields:
```

Comment:
there are some fields that could we removed at main entity
* last_edited
* is_active
* external_attendee_id
* external_order_id
* order_id
* qr_code
and at collections
* applied_taxes
* badge. print_excerpt